include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp

```diff
@@ -445,24 +445,115 @@ struct UniversalGemmPipelineAgBgCrPolicy
     }
 
     template <typename Problem>
-    CK_TILE_HOST_DEVICE static constexpr auto GetBlockGemm()
+    static auto GetBlockGemmType()
```

Comment:
since this is just a helper function, mark this **private**. and define another type alias for user.
```c++
private: static auto GetBlockGemm() {
    if constexpr(Problem::kBlockMethod == 0)
    {
        return custom_std::type_identity<decltype(GetBlockGemmUniversal<Problem>())>{};
    }
    else
    {
        return custom_std::type_identity<decltype(GetBlockGemmGlobalRegister<Problem>())>{};
    }
}

using BlockGemmType = typename decltype(GetBlockGemm())::type;
```